features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -51,7 +51,7 @@ internal class RumActionScope(
     private val networkInfo = sdkCore.networkInfo
 
     internal val attributes: MutableMap<String, Any?> = initialAttributes.toMutableMap().apply {
-        putAll(GlobalRumMonitor.get(sdkCore).getAttributes())
+        putAll(GlobalRumMonitor.get(sdkCore).getAttributes() + initialAttributes)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏ Why do you want to do this ? The `initialAttributes` are already there when `putAll` is called.